### PR TITLE
Add Examples to Docs

### PR DESCRIPTION
**Pull request by norton120**
Before submitting your PR, please review the following checklist:

- [x] Consider adding a unit test if your PR resolves an issue. N/A
- [x] All new and existing tests pass locally (`palm test`)
- [x] Lint (`palm lint`) has passed locally and any fixes were made for failures
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Breaking changes

- [ ] Check if this pull request introduces a breaking change

## What does this implement/fix? Explain your changes.
Added examples of palm use to the docs, and a viewpoint statement. 

## Does this close any currently open issues?
Nope

## Any other comments?
This is a living, growing document and should continue to evolve as we understand palm better. 

## Where has this been tested?

**Operating System:** MacOs

**Platform:** Catalina

**Target Platform:** All (readthedocs web)

### CODE REVIEW

File: docs/source/introduction/examples.rst [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+
+Palm & The Modern Work Dynamic
+==============================
+
+Depending on which sources you reference, tenure for a Software Developer is only 12-20 months (on average) at any given organization. 
+Regardless of the reason for this, the migratory nature of Software Development was one of the intial motivators for creating palm. 
+We simply became tired of building one-off dev toolkits every time we started a new role.
+
+With a rapidly changing
```
Comment (mrogers-palmetto) [inline docs/source/introduction/examples.rst:121]: Is this thought finished? And what goes under these sub-headers below?

File: docs/source/introduction/viewpoint.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
+================================
+Wait. Why Do My CLIs need a CLI?
+================================
+
+For a seasoned Engineer, the idea of wrapping public software interfaces in an abstraction layer will probably set off some warning bells. Admittedly, at first glace palm can appear to add unnecessary complexity and “magic” to an already polished API. For example, bringing up a docker-compose service stack:
```
Comment (mrogers-palmetto) [inline docs/source/introduction/viewpoint.rst:5]:
```suggestion
For a seasoned Engineer, the idea of wrapping public software interfaces in an abstraction layer will probably set off some warning bells. Admittedly, at first glance palm can appear to add unnecessary complexity and “magic” to an already polished API. For example, bringing up a docker-compose service stack:
```

File: docs/source/introduction/viewpoint.rst [hunk 3]
```diff
@@ -0,0 +1,131 @@
+================================
```
Comment (mrogers-palmetto) [inline docs/source/introduction/viewpoint.rst:1]: I absolutely ❤️  this doc

File: docs/source/introduction/viewpoint.rst [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+.. code::  
+
+   #Readme.md
+      
+   ## Troubleshooting
+   **“ERROR: unable to read file app/conf: file does not exist or access denied”** : 
+   your docker environment may be out of memory. Start by running `docker rm -f $(docker ps -qa)` … 
+
+But what if, instead of writing docs - docs the Developers will likely forget to check, 
+with steps that will need to methodically replicated,
```
Comment (mrogers-palmetto) [inline docs/source/introduction/viewpoint.rst:99]:
```suggestion
with steps that will need to be methodically replicated, 
```

File: docs/source/introduction/viewpoint.rst [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+with steps that will need to methodically replicated, 
+what if the fix was automated? Enter palm. 
+
+.. code:: python
+
+   #.palm/cmd_up.py
+
+   def cmd_up(ctx):
+       “”” starts the compose stack”””
+       echo(“starting docker stack…”)
```
Comment (mrogers-palmetto) [inline docs/source/introduction/viewpoint.rst:108]:
```suggestion
       echo(“Starting docker stack…”)
```

File: docs/source/introduction/viewpoint.rst [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+   /* Function for determining customer balance.
+       This function takes the current known balance, 
+       queries the bank api with the customer account,
+       then if the api returns a new balance it applies it to… (etc etc)
+   */ 
+   function customerBalance(custId){
+      // awful code goes here 
+   }
+
+And yet, the first instinct when engineering a development environments is
```
Comment (mrogers-palmetto) [inline docs/source/introduction/viewpoint.rst:76]:
```suggestion
And yet, the first instinct when engineering a development environment is 
```

File: docs/source/introduction/viewpoint.rst [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+   # same thing, via palm
+
+   $ palm test
+   Starting compose…
+   Unable to compose, stopping orphan container… Stopped.
+   Starting compose… Started.
+   Running 84 tests via pytest…
+
+Suddenly, palm starts to make a lot of sense. The abstraction layer that palm provides allows you to architect simplicity and consistency into your development environments, across languages, frameworks, and infrastructure designs. You decide what each command does, designing the workflow interface for each repo. A simple, single-container Jekyll app, and complex, multi-cloud microservices ecosystem, implemented by different teams with completely different skills, can share a common interface.
```
Comment (mrogers-palmetto) [inline docs/source/introduction/viewpoint.rst:52]:
```suggestion
Suddenly, palm starts to make a lot of sense. The abstraction layer that palm provides allows you to architect simplicity and consistency into your development environments across languages, frameworks, and infrastructure designs. You decide what each command does, designing the workflow interface for each repo. A simple single-container Jekyll app, and a complex multi-cloud microservices ecosystem, each implemented by different teams with completely different skills, can share a common interface. 
```

File: docs/source/introduction/viewpoint.rst (hunk 3 above)
Comment (mrogers-palmetto) [reply docs/source/introduction/viewpoint.rst:1]: This should be a Medium post, not just a doc file.

**Review by emekdahl-palmetto — APPROVED** [review]
I've reviewed this. Once the code suggestions are incorporated, I'm comfortable with merging.

File: docs/source/introduction/examples.rst (hunk 1 above)
Comment (ethanknox-palmetto) [reply docs/source/introduction/examples.rst]: ran out of gas here - so I'm clobbering it for now. will update soon :)

**Pull request merged**